<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editeur de niveau</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="CWEditor" name="wEditor" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>673</width>
              <height>673</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbMur">
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/mur.png</normaloff>:/image/mur.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbCroix">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/tCroix.png</normaloff>:/image/tCroix.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbDepart">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/tDepartFin.png</normaloff>:/image/tDepartFin.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbDQuart">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/tDQuart.png</normaloff>:/image/tDQuart.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pbHoriz">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/tHorizontal.png</normaloff>:/image/tHorizontal.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pbQuart">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lvledit.qrc">
              <normaloff>:/image/tQuart.png</normaloff>:/image/tQuart.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="cbBouge">
         <property name="text">
          <string>Bougeable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTourne">
         <property name="text">
          <string>Tournable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDepart">
         <property name="text">
          <string>Départ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbArrive">
         <property name="text">
          <string>Arrivée</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wRotation" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rb0">
            <property name="text">
             <string>&amp;0 °</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb90">
            <property name="text">
             <string>&amp;90 °</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb180">
            <property name="text">
             <string>&amp;180 °</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb270">
            <property name="text">
             <string>&amp;270 °</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CWEditor</class>
   <extends>QWidget</extends>
   <header>CWEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lvledit.qrc"/>
 </resources>
 <connections/>
</ui>
